aptos-move/aptos-vm-environment/src/prod_configs.rs

```diff
@@ -23,6 +23,7 @@ use move_vm_types::{
 use once_cell::sync::OnceCell;
 
 static PARANOID_TYPE_CHECKS: OnceCell<bool> = OnceCell::new();
+static PARANOID_REF_CHECKS: OnceCell<bool> = OnceCell::new();
```

Comment:
Not necessarily something to change in this PR, but I really think we should try to avoid using global variables as ways to config the VM.